src/presentation/index.ts

```diff
@@ -19,20 +20,30 @@ class SofieServer {
   public server: Express
 
   constructor() {
+    this.server = express()
```

Comment:
What's the reason for moving this line up here and not keep it in `configureServer`?

Reply:
Stack Overflow. In an issue the approved answer changed from express.json() to body parser.json() and then moved the this.server = express() up in the constructor. I assumed it had something to do with the lifecycle events and making sure express was instantiated early enough in the loop.

Reply:
The shouldn't be any difference from having it directly in the constructor than having it as the first thing in the first method of the constructor.

Personnaly, I would prefer that it got back into the method it was in, so we don't mix "abstraction levels". The constructor was nice and clean with just a single level before :)

Reply:
Yeah there is no difference in this instance :) But I like that you thought about lifecycle events! 😄 those are usually very important!

I agree with @LindvedKrvang about it just going back to configureServer here :) But good to know that you had a thought about this! I had to investigate it myself, so... good to be challenged on 😄

Reply:
Yep, you are completely right and had I taken a moment to watch the final layout of the code after moving stuff around I would have noticed that.

Alas, I did not. Moved it back now though.